src/lib/routes/admin-api/user-admin.ts

```diff
@@ -256,8 +318,13 @@ export default class UserAdminController extends Controller {
                 openApiService.validPath({
                     tags: ['Users'],
                     operationId: 'updateUser',
+                    summary: 'Update a user',
+                    description: 'Updates use with new fields',
```

Comment:
Are all fields required? Are non-submitted fields ignored? In other words: does this act more like a "true PUT" or a "PATCH"? It'd be good to mention that in the description.